packages/ui/src/2_molecules/WalletIdentity/WalletIdentity.module.css

```
@@ -4,4 +4,8 @@
 
 .dropdown {
   @apply w-40;
+
+  &Menu {
+    @apply p-3 bg-gray-90;
+  }
 }
```

```suggestion
.dropdown {
  @apply w-40;
}

.dropdownMenu {
  @apply p-3 bg-gray-90;
}
```
It's not necessary to write it like it was written and it's a bit confusing as well :D